src/main/main.cpp

```diff
@@ -527,6 +528,9 @@ int main(int argc, const char** argv) {
             fdtransfer = buf;
             params << ",fdtransfer=" << fdtransfer;
 
+        } else if (arg == "--target-cpu") {
+            params << "," << (arg.str() + 2) << "=" << args.next();
```

Comment:
You can merge this branch with other one-arg cases above.

Reply:
cba56c6